packages/widget-chat/src/actions/conversation.js

```diff
@@ -0,0 +1,27 @@
+export const CREATE_CONVERSATION = `CREATE_CONVERSATION`;
+export function createConversation(userId) {
+  return {
+    type: CREATE_CONVERSATION,
+    userId
+  };
+}
+
+export const RECEIVE_CONVERSATION = `RECEIVE_CONVERSATION`;
+export function receiveConversation(userId, conversation) {
+  return {
+    type: RECEIVE_CONVERSATION,
+    userId,
+    conversation
+  };
+}
+
+export function createConversationWithUser(userId, spark) {
+  return (dispatch) => {
+    dispatch(createConversation(userId));
+    const params = {
+      participants: [userId]
+    };
+    spark.conversation.create(params)
+      .then((conversation) => dispatch(receiveConversation(userId, conversation)));
```

Comment:
This is probably my fault, but are we talking about the user's uuid here or the `userId` value that's being stored? If it's the one that's being stored then it's actually the user Email. Can create conversation actually get by userEmail?
We should just adjust the var names so that that they identify what's being stored accurately.